src/sidebar.js

```diff
@@ -277,6 +277,9 @@ module.exports = {
       label: "Permissions and relationships",
       items: [
         "keto/index",
+        {
+          "Modeling Guides": ["keto/modeling/learn-the-concepts"],
+        },
```

Comment:
This kind of category isn't in the Notion plan. Instead, a `Create a permission model` doc. is exposed.

Reply:
It's here: https://www.notion.so/Learn-the-Concept-dbf112da3c884c5ba6f32151aefc3007